src/plugins/discover/public/application/apps/main/utils/resolve_index_pattern.ts

```diff
@@ -107,8 +107,8 @@ export function resolveIndexPattern(
   }
 
   if (stateVal && !stateValFound) {
-    const warningTitle = i18n.translate('discover.valueIsNotConfiguredIndexPatternIDWarningTitle', {
-      defaultMessage: '{stateVal} is not a configured index pattern ID',
+    const warningTitle = i18n.translate('discover.valueIsNotConfiguredDataViewIDWarningTitle', {
```

Comment:
I don't think we should be changing the keys just yet. I would leave this as a code-cleanup phase.

Reply:
I guess it's fine as in other renaming PRs keys were updated too